openml-lightgbm/lightgbm-provider/src/main/java/com/feedzai/openml/provider/lightgbm/LightGBMExplainer.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2022 Feedzai
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.feedzai.openml.provider.lightgbm;
+
+import com.feedzai.openml.data.Instance;
+import com.feedzai.openml.explanations.ModelExplainer;
+
+/**
+ * A feature contribution provider for the predictions of the {@link LightGBMBinaryClassificationModel}.
+ */
+public class LightGBMExplainer implements ModelExplainer<LightGBMBinaryClassificationModel> {
+    /**
+     * The {@link LightGBMBinaryClassificationModel} whose predictions will be explained.
+     */
+    private final LightGBMBinaryClassificationModel model;
+
+    /**
+     * Constructor.
+     *
+     * @param model The {@link LightGBMBinaryClassificationModel} whose predictions will be explained.
+     */
+    public LightGBMExplainer(final LightGBMBinaryClassificationModel model) {
+        this.model = model;
+    }
+
+    @Override
+    public double[] getFeatureContributions(final Instance instance) {
+        return model.getFeatureContributions(instance);
+    }
```

Comment:
even though this is just a wrapper for now, should we add tests for it ?

btw should we deprecate the method in the LightGBMBinaryClassificationModel, like we did with Fdz Random forest?

Reply:
We are, in `com/feedzai/openml/provider/lightgbm/LightGBMBinaryClassificationModelTrainerTest.java:361`
But I will also adapt Sheng's latest test anyway

Reply:
> btw should we deprecate the method in the LightGBMBinaryClassificationModel, like we did with Fdz Random forest?
IMO no, because it is being used by this method and it must be. Since the feature contributions are implemented in C++, which we access through the SWIG. 
We could also expose the SWIG in LightGBMBinaryClassificationModel, but I believe it is simpler to just have this  method wrapper

Reply:
I agree but I think this will cause users of the model to keep using the old method, no ?

Not sure if deprecating is what I wanted to say because I agree that here we should keep using it the way it is... but maybe edit the javadoc of the original method to discourage its use and point to this one instead?

Reply:
You probably didn't notice that I changed the method to protected